src/main/java/com/smartcar/sdk/data/BatchResponse.java

```diff
@@ -38,13 +40,14 @@ private static String toCamelCase(String fieldName) {
 
   public <T extends ApiData> SmartcarResponse<T> get(String path) throws SmartcarException {
     JsonObject res = this.responseData.get(path);
-
     Integer code = res.get("code").getAsInt();
+    JsonObject body = res.get("body").getAsJsonObject();
+
     if (code != 200) {
-      // TODO : handle error
-      System.out.println("ERROR");
+      String errorMessage = body.get("message").getAsString();
+      throw new SmartcarException(errorMessage);
```

Comment:
We should use the longer constructor or our Factory to parse the errorCode/statusCode, ...
```java
public SmartcarException(int statusCode, String error, String message, String code)
```
or 
```java
  public static SmartcarException Factory(final Response response) throws IOException {
```

Take a look here: https://github.com/smartcar/java-sdk/blob/master/src/main/java/com/smartcar/sdk/SmartcarException.java#L22